SwiftHEXColorsExamples.xcodeproj/project.pbxproj

```diff
@@ -115,6 +115,7 @@
 				TargetAttributes = {
 					846C69C419DFF12F00AB429E = {
 						CreatedOnToolsVersion = 6.0.1;
+						DevelopmentTeam = 5B74ZU8KV8;
```

Comment:
Could you please remove this and other `DevelopmentTeam` build settings that you added?